### DISCUSSION TITLE
ArcGIS Module v4.1.0 Released!

### DISCUSSION THREAD

**Discussion by cameronkroeker** [Esri/arcgis-powershell-dsc#461] [Announcements]
We are excited to announce that ArcGIS Module v4.1.0 has been released to GitHub and PowerShell Gallery.

#### What's New and ArcGIS Module download
https://github.com/Esri/arcgis-powershell-dsc/releases/tag/v4.1.0

#### Samples
https://github.com/Esri/arcgis-powershell-dsc/tree/main/SampleConfigs/v4/v4.1.0

#### Configuration JSON variable reference page
https://github.com/Esri/arcgis-powershell-dsc/wiki/v4.1.0-Variables-reference-page-for-JSON-configuration-files

**Comment by JunoEDK** [comment]
Hi,
Thanks for your work. Is it possible to download arcgis extension using this script?

**Reply by cameronkroeker** [reply]
Yes, the module can download ArcGIS Server extensions, and ArcGIS Pro Data Interoperability extension, and ArcGIS Desktop extensions.
